metricshub-http-extension/src/main/java/org/sentrysoftware/metricshub/extension/http/HttpExtension.java

```diff
@@ -116,7 +103,7 @@ public void checkProtocol(TelemetryManager telemetryManager) {
 
 		// Stop the HTTP health check if there is not an HTTP configuration
 		if (httpConfiguration == null) {
-			return;
+			return false;

```

Comment:
This introduces a regression. If the protocol is not configured the protocol check cannot be reported as false (down). In that case it is better to return an `Optional<Boolean>` instance rather than primitive `boolean`.